packages/form-schema/src/components/CheckboxesWidget.js

```
@@ -0,0 +1,61 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/no-autofocus */
```

I added these lines to ignore the errors I was getting